inc/compat/themes/compat-theme-minimog.php

```diff
@@ -22,6 +22,16 @@ public function hooks() {
 		// Scripts
 		add_action( 'wp_enqueue_scripts', array( $this, 'deregister_woocommerce_scripts' ), 20 );
 		add_action( 'wp_enqueue_scripts', array( FluidCheckout_Enqueue::instance(), 'replace_woocommerce_scripts' ), 20 );
+
+		// Remove checkout payment info heading
+		add_filter( 'fc_content_section_class', array( $this, 'remove_checkout_payment_heading' ), 20 );
+	}
+
+	 /**
+	 * remove checkout payment info heading added by the theme
+	 */
+	public function remove_checkout_payment_heading() {
+		remove_action( 'woocommerce_checkout_after_order_review',array( Minimog\Woo\Checkout::instance(), 'template_checkout_payment_title' ),10 );
 	}
```

Comment:
This function seems unnecessary, as the hook related functions such as `remove_action` should always be called from within one of the `hooks()` functions, as explained above.